README.md

```diff
@@ -11,33 +11,41 @@ Add our SDK as a package dependency to your XCode project [with the following st
 To start an upload, you must first create an [upload URL](https://docs.mux.com/guides/video/upload-files-directly). Then, pass the upload URL and the file to be uploaded into the SDK.
 
 ```swift
-let upload = MuxUpload.Builder(uploadURL: URL(string: myUploadURL)!, videoFile: videoFile)
-    .withMIMEType(type: "video/*")
-    .build()
+let upload = MuxUpload(
+    uploadURL: URL(string: PUT_URL)!,
+    videoFileURL: videoFile,
+    videoMIMEType: "video/*"
```

Comment:
Can we make this the default MIMEType so we don't have to pass it in as a param? I don't think we'll ever need to configure this for a MuxUpload.

Reply:
Looking at the code, looks like this is the default value for this param https://github.com/muxinc/swift-upload-sdk/blob/main/Sources/MuxUploadSDK/Public%20API/MuxUpload.swift#L56

I recommend we remove it entirely to simplify the example.